types/react-instantsearch-native/index.d.ts

```diff
@@ -5,7 +5,7 @@
 //                 Haroen Viaene <https://github.com/haroenv>
 //                 Samuel Vaillant <https://github.com/samouss>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.9
+// TypeScript Version: 3.2
```

Comment:
is this change unrelated?

Reply:
No, dtslint told me to do it after the changes I made. I must be using something that was added/fixed in 3.2.

Reply:
Interesting, as far as I can tell react-autosuggest isn't a dependency

Reply:
It's a dependency in one of the react-instantsearch-core tests, https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-instantsearch-core/react-instantsearch-core-tests.tsx#L442

I included that test because of the example in the Algoliasearch documentation here: https://github.com/algolia/react-instantsearch/blob/master/examples/autocomplete/src/App-Multi-Index.js

We could remove that test then we don't have to change the Typescript version in these files.  What changing the TypeScript version does is it requires people to use Typescript 3.2 in order to use our package.

Reply:
Thanks—done!